paparazzi/src/main/java/app/cash/paparazzi/internal/resources/RepositoryLoader.kt

```diff
@@ -927,15 +927,12 @@ abstract class RepositoryLoader<T : LoadableResourceRepository>(
   }
 
   protected fun getResRelativePath(file: PathString): String {
-    if (file.isAbsolute) {
+    if (file.isAbsolute || !isLoadingFromZipArchive) {
```

Comment:
```suggestion
    if (file.isAbsolute) {
```

Reply:
`isLoadingFromZipArchive` is needed here otherwise `../module/build/intermediates/packaged_res/debug/` cannot be handled as there is no `res` segment in the path. It is tricky one. Let us discuss in next sync meeting. Or do you have better suggestion?